adops/adops-planning-guide.md

```diff
@@ -63,6 +63,28 @@ Some additional resources that might be helpful as you work through your setup i
 
 [Deals in Prebid](/adops/deals.html): You can negotiate deals with header bidding demand partners and have them compete with ad server inventory. You’ll need to create additional line items to support these deals. This document explains how to set up Prebid to make that happen.
 
+## Special Considerations
+
+### Rendering mobile outstream ads
+
+The 'outstream' format requires different ways of rendering for Prebid.js vs Mobile:
+
+- For Prebid.js web integrations, outstream ads are rendered like banners.
+- For Prebid Mobile integrations, outstream must be rendered similar to instream video.
+
+This difference has implications on the ad server setup; if not done carefully, it
+can cause the ad server to choose the wrong line item and cause the creative to
+not render properly.
+
+Prebid's recommendation is to
+consider setting up outstream adunits as slightly different sizes.
```

Comment:
Should this recommendation not apply for publishers who have a player capable of rendering outstream ?

Reply:
If they're using outstream, they've defined a renderer that can play VAST. But the problem is in the ad server line item setup.